test/functional/job_configs.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, config, expected_to_fail, expected_num_subjobs, expected_num_
     teardown_build:
         - echo "Doing build teardown."
         - sleep 1
-        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt)
+        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt || mktemp -t clusterrunner)
```

Comment:
If there are no subjob_file_*.txt files, the ls command will exit non-zero and the whole build fails immediately.  Not what I expected from ls.